sentry_sdk/integrations/flask.py

```diff
@@ -44,6 +44,8 @@
 except ImportError:
     raise DidNotEnable("blinker is not installed")
 
+installed_packages = _get_installed_modules()
+FLASK_VERSION = installed_packages("flask")
```

Comment:
`_get_installed_modules()` returns a dict, so you'd want:
```suggestion
FLASK_VERSION = installed_packages["flask"]
```
I'm also thinking we could move these two lines into the `setup_once` method instead, so that all the version checking code is in one place.